src/pages/NewDiveForm.js

```diff
@@ -227,3 +291,23 @@ const MapButton = styled(NavLink)`
   font-size: 1.5rem;
   color: orange;
 `;
+
+const ImageUpload = styled.div`
+  grid-column-start: 1;
+  grid-column-end: 3;
```

Comment:
Maybe you can have a look at grid-template-areas. Not necesserily here, but it can make styling with areas a lot easier.